web/src/compose_validate.js

```diff
@@ -750,35 +750,26 @@ export function check_overflow_text() {
     // expensive.
     const text = compose_state.message_content();
     const max_length = realm.max_message_length;
+    const remainingCharacters = max_length - text.length;
```

Comment:
If you observe carefully the other variable names here, you'll see we use snake_case and not camelCase. You can just rename this to `remaining_length` and avoid assigning it values when rendering the template below

Reply:
ok I will rename the variable as suggested